src/home.js

```diff
@@ -0,0 +1,36 @@
+export default function homePage() {
```

Comment:
- [ ] It would be great if you use ES6 syntax here. For example, don't export a function before defining a function and use arrow functions.